package.json

```diff
@@ -64,6 +64,7 @@
     "test": "sh scripts/lint.sh && sh scripts/tests.sh",
     "test:server": "sh scripts/tests.sh",
     "lint": "sh scripts/lint.sh",
+    "prewatch:client": "cd client && npm install",
```

Comment:
If I understand correctly, ```npm install``` will be run for each ```watch``` script? Is it necessary? I think it's good for the build script but maybe it's too much for the dev.